A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BAT/A320_Neo_BAT.js

```diff
@@ -29,7 +29,7 @@ var A320_Neo_BAT;
                 }
             } else {
                 for (let i = 0; i < 2; ++i) {
-                    const batValue = SimVar.GetSimVarValue("ELECTRICAL BATTERY BUS VOLTAGE", "Volts");
+                    const batValue = SimVar.GetSimVarValue(`L:A32NX_ELEC_BAT_${i + 10}_POTENTIAL`, "Volts");
```

Comment:
What's the reason for BAT_10 and BAT_11 over BAT_1 and BAT_2 like e.g. IDG?

Reply:
`A320_NEO_INTERIOR.xml` defines `BATTERY_BUS_ID` as such (10 and 11). I might replace this with 1 and 2 in a PR which replaces the default ELEC system.